### DISCUSSION TITLE
Issue on pulling the docker and accessing it

### DISCUSSION THREAD

**Discussion by nk8291** [apache/superset#24464] [Q&A / Help]
Dear Community,

I have cloned the git and deployed the superset on centos server. After this over development team has created multiple dashboard and published it in superset portal.

Now we have done some changes in branding logos and executed the below commands for accessing the superset:

cd superset
docker-compose -f docker-compose-non-dev.yml down
docker-compose -f docker-compose-non-dev.yml pull
docker-compose -f docker-compose-non-dev.yml up -d

so when we accessed the superset portal, all charts are present and accessible. also I can see the list of dashboard created by developer. but when I try to open I get below error:

Unexpected error
Unexpected error: TypeError: Cannot convert undefined or null to object

    in wu
    in div
    in ForwardRef
    in div
    in Styled(div)
    in div
    in Styled(div)
    in Eu
    in vh
    in d
    in div
    in Styled(div)
    in div
    in Styled(div)
    in xh
    in div
    in Styled(div)
    in Jh
    in ep
    in Connect(ep)
    in A
    in n
    in d
    in Suspense
    in t
    in t
    in b
    in t
    in P
    in E
    in d
    in lt
    in Unknown
    in s
    in y
    in bt
    in t
    in t
    in Unknown

![superset_error](https://github.com/apache/superset/assets/62594274/4fa65a93-abe3-4765-90b8-16be67b2d2c9)

Can you please how to fix this asap???

**Comment by nk8291** [comment]
Can any one suggest what would be the fix for this

**Comment by sebastianliebscher** [comment]
Hi @nk8291! Your issue title is misleading. This is a customizing question rather than a bug. You can join [Superset Slack](http://bit.ly/join-superset-slack) to ask for help in #customizing-superset. There you can provide more context.

**Comment by nk8291** [comment]
Hi @sebastianliebscher , I have not done any customizations. Below are the steps followed for the setup and deployment:

1. Git clone Superset Repo
2. cd superset
3. docker-compose -f docker-compose-non-dev.yml pull
4. docker-compose -f docker-compose-non-dev.yml up -d
5. Access superset on server-ip:8088/login
6. Developed multiple charts/ dashboards 
7. once again, followed below steps(as there are some server activities happening so we had to run the docker service again), but this we followed below steps as well:
- cd superset
- docker-compose -f docker-compose-non-dev.yml down 
- docker-compose -f docker-compose-non-dev.yml pull
- docker-compose -f docker-compose-non-dev.yml up -d

after this we tried to access the dashboards in superset portal, and faced the attached issue. Although all the charts, dataset, database connection, users, roles are fine only dashboards were impacted and given issues occurred.

So, I Suspect this must be related to re-deployment of the application/ restarting the docker container of the application related issue.

**Comment by sebastianliebscher** [comment]
Can you share relevant parts of your docker-compose logs as I am not able to reproduce this issue with the provided steps?
